frontend/Pages/ConversationsFeed/index.tsx

```diff
@@ -136,7 +136,7 @@ export const ConversationsFeed: React.FC = () => {
           </View>
           <View style={styles.contactInfo}>
             <View style={styles.contactName}>
-              <Text>{moment.unix(item.created_at).format('L HH:mm')}</Text>
+              <Text>{formatDate(item.created_at, 'L HH:mm')}</Text>
```

Comment:
it looks like here is not formatting correctly, it displays dates like `1 09:59`